calculator/parser.py

```diff
@@ -40,49 +51,94 @@ def binary_operator(lhs, rhs, op):
         )
 
 
-def push(e, stack):
+def is_numeric_token(token):
+    """
+    Says whether or not the token input is a number
+
+    Parameter
+    ---------
+    token : token
+        The token to be evalueated
+
+    Returns
+    -------
+    bool: Whether or not this token is a numeric character
     """
-    Pushes the element (e) to the given stack, which mutates the stack
 
+    return isinstance(token, numbers.Number)
+
+
+def push(value, stack):
+    """
+    Pushes the element (value) to the given stack, which mutates the stack
     Parameters
     ----------
-    e : element
+    value : element
         The element to be pushed on top of the stack
-
     stack : list
         The stack that e is going to be pushed onto
-
     """
-    stack.append(e)
+    stack.append(value)
 
 
 def pop(stack):
     """
     Retrieves the last element from the given stack and deletes it, which mutates it
-
-    Parameters
     ----------
```

Comment:
This docstring is wrong.  You're missing the `Parameters` here and you're not indicating what it returns.